rebar.config

```diff
@@ -10,7 +10,8 @@
         folsom,
         jsx,
         dns_erlang,
-        iso8601
+        iso8601,
+	{nodefinder, "1.8.0"}
```

Comment:
Is there a reason you are pinning to 1.8.0 of nodefinder instead of the 2.0.0 version that was released on June 4th, 2020?

Reply:
No there is no specific reason for that, I just didn't realize there was a new version out.
Will upgrade